debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -319,6 +319,14 @@ public ValueConverter converter(Column column, Field fieldDefn) {
         return super.converter(column, fieldDefn);
     }
 
+    @Override
+    protected Object handleUnknownData(Column column, Field fieldDefn, Object data) {
```

Comment:
@nicholas-fwang Looking at it I believe the best solution is to modify `io.debezium.jdbc.JdbcValueConverters.handleUnknownData(Column, Field, Object)` instead of adding the method to MySQL only.  All handling should be ideally done in `createValueGenerator`.
It is just necessary to think about how to modify the logic to keep it backward compatible for the default setting.

Reply:
@jpechane okay, I will make throw exception in `io.debezium.jdbc.JdbcValueConverters.handleUnknownData`.
and for backward compatible, the default setting should be `SKIP` or `IGNORE`, not `FAIL`. I'll also update it.